components/sample/keras/train_classifier/README.md

```diff
@@ -0,0 +1,20 @@
+## Train classification model using Keras ##
+
+Usage:
+
+~~~~
+#Load the component
+train_op = comp.load_component(url='https://raw.githubusercontent.com/Ark-kun/pipelines/Added-sample-component/components/sample/keras/train_classifier/component.yaml')
+
+#Use the component as part of the pipeline
+def pipeline():
+    train_task = train_op(
+        training_set_features_path=os.path.join(testdata_root, 'training_set_features.tsv'),
```

Comment:
The signature of the `train_op` function corresponds to the inputs section of the component.
Some tweaks are performed to make the signature valid and pythonic:
1) Inputs with default values come after the inputs without default values
2) Input names are converted to pythonic names (spaces and symbols replaced with underscores and letters lowercased).

I'd encourage component creators to use "human names" for inputs and outputs.
But in interest of reducing confusion I'll change the input names in component definition to pythonic names, so that they match the python function signature.

Reply:
Done.